package.json

```
@@ -16,6 +16,7 @@
     "@redhat-cloud-services/frontend-components-utilities": "4.0.19",
     "@reduxjs/toolkit": "2.3.0",
     "@scalprum/react-core": "0.9.3",
+    "@sentry/webpack-plugin": "^2.22.6",
```

Can you please pin the version? Otherwise looks good and I'm ready to approve whenever the sentry side of things is ready.

```suggestion
    "@sentry/webpack-plugin": "2.22.6",
```